src/JeffreyDB.ts

```diff
@@ -0,0 +1,98 @@
+import { Sequelize, DataTypes, Model } from 'sequelize';
+
+export const sequelize = new Sequelize('database', 'user', 'password', {
```

Comment:
Have users specify database name, username, password in `.env` file instead of hardcoding.

Look into this and see if you need to provide this in the first place